synopsys-task/src/synopsys-task/synopsys-bridge.ts

```diff
@@ -422,6 +435,44 @@ export class SynopsysBridge {
     return false;
   }
 
+  async getSynopsysBridgePath(): Promise<string> {
+    let synopsysBridgePath = inputs.SYNOPSYS_BRIDGE_PATH;
+
+    if (!synopsysBridgePath) {
+      synopsysBridgePath = this.getBridgeDefaultPath();
+    }
+    return synopsysBridgePath;
+  }
+
+  async getVersionFromLatestURL(): Promise<string> {
+    try {
+      const latestVersionsUrl = this.bridgeArtifactoryURL.concat(
+        "latest/versions.txt"
+      );
+      const httpClient = new HttpClient("");
+      const httpResponse = await httpClient.get(latestVersionsUrl, {
+        Accept: "text/html",
+      });
+      if (httpResponse.message.statusCode === 200) {
+        const htmlResponse = (await httpResponse.readBody()).trim();
+        const lines = htmlResponse.split("\n");
+        for (const line of lines) {
+          if (line.includes("Synopsys Bridge Package")) {
+            const newerVersion = line.split(":")[1].trim();
+            return newerVersion;
+          }
+        }
+      } else {
+        taskLib.error("Unable to locate version url");
```

Comment:
can we make more relatable error message like: Unable to fetch the latest version from artifactory
we can check once with team.